test/unit/import_playbooks_test.rb

```diff
@@ -0,0 +1,74 @@
+require 'test_plugin_helper'
+require_relative '../../app/services/foreman_ansible/playbooks_importer'
+require_relative '../fixtures/output_example_from_playbooks_importer'
+
+class ImportPlaybooksTest < ActiveSupport::TestCase
+  setup do
+    @actual_importer = ::ForemanAnsible::PlaybooksImporter.new(@proxy)
```

Comment:
this could be initialized outside of the setup block in a special `subject` block I'd imagine it something like:
```ruby
let(:ansible_proxy) { FactoryBot.create(:smart_proxy, :with_ansible) }
let(:proxy_api) { mock('ProxyAPI') } # this could even be a real thing by { ::ProxyAPI::Ansible.new(url: ansible_proxy.url }
subject { ::ForemanAnsible::PlaybooksImporter.new(ansible_proxy) }

setup do
  subject.stubs(:proxy_api).returns(proxy_api)
end

test '1' do
  proxy_api.stubs(:playbooks).returns([1,2]) # you probably want to return something useful ^_^
  assert_equal subject.playbooks.first, 1 # you probably want to have more elaborate test ^_^
  # .... test stuff..
end
```